applications/StatisticsApplication/README.md

```diff
@@ -0,0 +1,173 @@
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=\color{White}{\underline{r}&space;=&space;\frac{1}{T_{total}}\sum_{i=1}^{n}{\underline{x}\Delta&space;t_i}&space;\quad&space;where&space;\quad&space;T_{total}&space;=&space;T_{end}&space;-&space;T_{initial}&space;\quad&space;and&space;\quad&space;\Delta&space;t_i&space;=&space;T_{i}&space;-&space;T_{i-1}&space;\quad&space;\forall&space;T_i&space;\in&space;\left\lbrace&space;T_{initial},&space;...,&space;T_{end}&space;\right\rbrace}" target="_blank"><img src="https://latex.codecogs.com/svg.latex?\color{White}{\underline{r}&space;=&space;\frac{1}{T_{total}}\sum_{i=1}^{n}{\underline{x}\Delta&space;t_i}&space;\quad&space;where&space;\quad&space;T_{total}&space;=&space;T_{end}&space;-&space;T_{initial}&space;\quad&space;and&space;\quad&space;\Delta&space;t_i&space;=&space;T_{i}&space;-&space;T_{i-1}&space;\quad&space;\forall&space;T_i&space;\in&space;\left\lbrace&space;T_{initial},&space;...,&space;T_{end}&space;\right\rbrace}" title="\color{White}{\underline{r} = \frac{1}{T_{total}}\sum_{i=1}^{n}{\underline{x}\Delta t_i} \quad where \quad T_{total} = T_{end} - T_{initial} \quad and \quad \Delta t_i = T_{i} - T_{i-1} \quad \forall T_i \in \left\lbrace T_{initial}, ..., T_{end} \right\rbrace}" /></a>
+
+Following is an example of mean calculation of non historical velocity. Input variable is node's non-historical container's `VELOCITY` and output variable is same containers `VELOCITY_MEAN` where mean will be stored for each node. The `0` represents echo level for this method object. Blank "" indicates that value method is used.
+
+```python
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.StatisticsApplication as KratosStats
+model = Kratos.Model()
+model_part = model.CreateModelPart("test_model_part")
+mean_method = KratosStats.TemporalMethods.NonHistorical.Nodes.ValueMethods.Mean.Array(model_part, "", Kratos.VELOCITY, 0, KratosStats.VELOCITY_MEAN)
+mean_method.InitializeStatisticsMethod()
+previous_time = 2
+for t in range(3, 6):
+    delta_time = t - previous_time
+    mean_method.CalculateStatistics(delta_time)
+    previous_time = t
+```
+
+#### Root mean square
+
+In the case of spatial domain, it calculates root mean square of a given variable for a given container and returns it as shown in following equation. x<sub>i</sub> is the i<sup>th</sup> element's variable value of the corresponding container. Result will have the same type as the type of the variable specified by the user. (If it has higher dimension than a scalar, mean of each dimension will be calculated seperately resulting with a mean having same dimension as the input dimension)
```

Comment:
Perhaps copy paste error or you refer to root mean square simply as mean, I would use the abbreviation rms to avoid confusion.